custom/ilsgateway/api.py

```diff
@@ -148,6 +148,9 @@ def prepare_commtrack_config(self):
         - Sets the proper location types hierarchy on the domain object.
         - Sets a keyword handler for reporting receipts
         """
+        for location_type in LocationType.objects.by_domain(self.domain):
```

Comment:
It's only called on the initial sync so it's should be good to delete all location types.